app/templates/errors.hbs

```diff
@@ -57,13 +57,13 @@
       <AuDataTableThSortable @label={{da}} @field='datetime' @currentSorting={{this.sort}} />
       <AuDataTableThSortable @label={{in}} @field='specificInformation' @currentSorting={{this.sort}} />
     </c.header>
-    <c.body as |row|>
-      <td>{{row.logLevel.label}}</td>
-      <td>{{row.logSource.label}}</td>
-      <td>{{last-part-uri row.className}}</td>
-      <td>{{row.message}}</td>
-      <td>{{format-plain-date row.datetime 'd MMMM yyyy, HH:mm'}}</td>
-      <td>{{row.specificInformation}}</td>
+    <c.body as |logEntry|>
```

Comment:
Not really needed, but while debugging some deprecation messages I found this code confusing, so renamed it to something more clear.